website/docs/releases/0_16_1.md

```diff
@@ -31,7 +31,7 @@ sidebar_position: 9975
 * Airflow: enforce column casing in `SQLCheckExtractor`s [`#1159`](https://github.com/OpenLineage/OpenLineage/pull/1159) [@denimalpaca](https://github.com/denimalpaca)  
     *Uses the parent extractor's `_is_uppercase_names` property to determine if the column should be upper cased in the `SQLColumnCheckExtractor`'s `_get_input_facets()` method.*
 * Spark: prevent exception when no schema provided [`#1180`](https://github.com/OpenLineage/OpenLineage/pull/1180) [@pawel-big-lebowski](https://github.com/pawel-big-lebowski)  
-    *Prevents evaluation of column lineage when the `schemaFacet` is `null`.*
+    *Prevents evaluation of column lineage when the `schemFacet` is `null`.*
```

Comment:
?

Reply:
[    *Prevents evalution of column lineage when the `schemFacet` is `null`.*](https://github.com/OpenLineage/docs/blob/4722a53119c84cc62677fa5cab6ff1467cbb7b3a/docs/releases/0_16_1.md?plain=1#L34)